src/System.Security.Cryptography.Pkcs/tests/AlgorithmIdentifierTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Pkcs.Tests
+{
+    public static class AlgorithmIdentifierTest
+    {
+        [Fact]
+        public static void ParameterlessConstructor()
+        {
+            AlgorithmIdentifier ai = new AlgorithmIdentifier();
+            Assert.Equal(0, ai.KeyLength);
+            Assert.Equal(Oids.TripleDesCbc, ai.Oid.Value);
+            Assert.Equal(typeof(byte), ai.Parameters.GetType().GetElementType());
```

Comment:
This type-test seems pretty well covered by the field being declared a `byte[]`.

It's implicitly an `Assert.NotNull` (since null would throw on GetType()), so maybe that's a better statement.